admin/con-rbac-config-permission-policies-external-file.adoc

```diff
@@ -77,16 +91,17 @@ data:
 ** *name*: `rbac-policy`
 --
 
-. Update the policy path in the `app-config.yaml` file as follows:
```

Comment:
IBM style uses "as follows" and "For example" as an accepted way to introduce code examples. Most of these types of lead-in statements currently follow this format throughout the RHDH docs.